internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -1137,6 +1137,14 @@ func upgradeSettingsSchema() *pluginsdk.Schema {
 						Type:     pluginsdk.TypeString,
 						Required: true,
 					},
+					"drain_timeout_in_minutes": {
+						Type:     pluginsdk.TypeInt,
+						Optional: true,
+					},
```

Comment:
According to the docs It looks like this has a default value of 30 minutes, so we should set that here as well. Is there also some validation that we could add? e.g. value must be between 0 and 60 minutes?